src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -61,9 +62,11 @@
     <Compile Include="System\Net\Sockets\SocketOptionName.cs" />
     <Compile Include="System\Net\Sockets\SocketShutdown.cs" />
     <Compile Include="System\Net\Sockets\TCPClient.cs" />
+    <Compile Condition="'$(TargetGroup)' != 'netcore50'" Include="System\Net\Sockets\TCPClient.netstandard17.cs" />
```

Comment:
Can we group these into one ItemGroup with the condition on it instead of duplicating the conditions multiple places.